src/record/internal/matroska_read.cpp

```diff
@@ -193,7 +193,7 @@ k4a_result_t parse_mkv(k4a_playback_context_t *context)
                     if (check_element_type(e, &seek))
                     {
                         KaxSeekID &seek_id = GetChild<KaxSeekID>(*seek);
-                        EbmlId ebml_id(seek_id.GetBuffer(), (const unsigned int)seek_id.GetSize());
+                        EbmlId ebml_id(seek_id.GetBuffer(), (unsigned int)seek_id.GetSize());
```

Comment:
GCC Warning: `forming offset 16 is out of the bounds [0, 15]`

Reply:
Since this is cpp should we use `static_cast` instead?

```suggestion
                        EbmlId ebml_id(seek_id.GetBuffer(), static_cast<unsigned int>(seek_id.GetSize()));
```